lib/foreman/renderer/unsafe_mode_renderer.rb

```diff
@@ -0,0 +1,17 @@
+module Foreman
+  module Renderer
+    class UnsafeModeRenderer < BaseRenderer
+      def render
+        erb = ERB.new(source_content, nil, '-')
+        erb.filename = source_name
```

Comment:
this dropped following
```ruby
# erb allows to set location since Ruby 2.2
erb.location = template_name, 0 if erb.respond_to?(:location=)
```

which I think is no longer needed since we always use new renderer even for snippets inside snippets, correct?